clis/teliod/src/nc.rs

```diff
@@ -86,12 +86,15 @@ struct NCConfig {
 }
 
 impl NotificationCenter {
-    pub async fn new(config: &super::TeliodDaemonConfig) -> Result<Self, Error> {
+    pub async fn new(
+        config: &super::TeliodDaemonConfig,
+        hw_identifier: &Uuid,
+    ) -> Result<Self, Error> {
         let callbacks = Arc::new(Mutex::new(vec![]));
 
         let nc_config = NCConfig {
             authentication_token: config.authentication_token.clone(),
-            app_user_uid: config.app_user_uid,
+            app_user_uid: *hw_identifier,
```

Comment:
nit: maybe rename `NCConfig`'s `app_user_uid` to `hw_id`?

Reply:
From MQTT API point of view this field is still called `app_user_uid`. I could change the `NCConfig` but `TokenHttpRequest`would still have to remain `app_user_uid` otherwise it would break the HTTP request.

I would rather change the `new()` signature.